static/js/tools/choropleth/choropleth.tsx

```diff
@@ -79,14 +79,22 @@ class ChoroplethMap extends Component {
    */
   loadValues(): void {
     let baseUrl = "/api/choropleth/values";
```

Comment:
Is there a way of not having this hard-coded?

Reply:
Hmm not that I know of, but I am sure that there might be one.